apps/marketplace/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import { Telegram } from "@mui/icons-material";
+import { Box, Grid, Link, Typography, useTheme } from "@mui/material";
+import { DiscordLogo, NEWMLogo, XLogo } from "@newm-web/assets";
+import { FunctionComponent } from "react";
+import {
+  NEWM_DISCORD_URL,
+  NEWM_IO_URL,
+  NEWM_MARKETPLACE_FAQ_URL,
+  NEWM_MARKETPLACE_SUPPORT_URL,
+  NEWM_MARKETPLACE_TERMS_OF_SERVICE_URL,
+  NEWM_TELEGRAM_URL,
+  NEWM_X_URL,
+} from "../../common/constants";
+
+const Footer: FunctionComponent = () => {
+  const theme = useTheme();
+
+  return (
+    <Grid
+      px={ [2, 5, 10] }
+      py={ [2.5, 3] }
+      sx={ {
+        alignItems: "center",
+        borderTop: `1px solid ${theme.colors.grey500}`,
+        display: "flex",
+        flexDirection: ["column", "row"],
+        gap: 2,
+        justifyContent: "space-between",
+      } }
+    >
+      <Box
+        sx={ {
+          alignItems: "center",
+          display: "flex",
+          flexDirection: ["column", "row"],
+          gap: [2, 4],
+        } }
+      >
+        <Link href={ NEWM_IO_URL } rel="noopener" target="_blank">
+          <NEWMLogo height="40" width="40" />
+        </Link>
+        <Typography
+          component="div"
+          sx={ {
+            alignItems: "center",
+            display: "flex",
+            flexDirection: ["column", "row"],
```

Comment:
I made the Marketplace links a column as they didn't seem to center well, I thought they threw off the look. This can be updated as we do a more thorough Mobile responsive review with actual content being displayed.